pkg/acceptance/cluster/dockercluster.go

```diff
@@ -177,32 +177,30 @@ func CreateDocker(
 
 	clusterID := uuid.MakeV4()
 	clusterIDS := clusterID.Short()
-	// Only pass a nonzero logDir down to DockerCluster when instructed to keep
-	// logs.
-	var uniqueLogDir string
-	logDirRemovable := false
-	if logDir != "" {
+
+	if volumeDir == "" {
+		volumeDir, err = ioutil.TempDir("", fmt.Sprintf("cockroach-acceptance-%s", clusterIDS))
+		maybePanic(err)
+	} else {
+		volumeDir = fmt.Sprintf("%s-%s", volumeDir, clusterIDS)
```

Comment:
So you don't actually use the `volumeDir` that was supplied? Perhaps just use `filepath.Join(volumeDir, clusterIDS)`.

Reply:
Mind changing it and if something goes wrong, you get to point the finger at me 👈 ?

Reply:
Works for me. Done!